### DISCUSSION TITLE
Jail breaking for iOS? Why?

### DISCUSSION THREAD

**Discussion by Fluffy728** [Impostor/Impostor#577] [Q&A] [answered]
Why do we need to jailbreak iOS to access imposter? But with cloud hosting you don’t have to, and that is probably illegal . 

refrence: https://serverflex.io/company/legals-and-statements/data-retention-policy

**Comment by miniduikboot** [comment, marked as answer]
This is not our site, and we don't control or endorse the content of that website.

Please refer to our region installation site for instructions on how to add an Impostor server to your menu (bonus points: it does not require a jailbreak on iOS)
https://impostor.github.io/Impostor/

**Reply by Fluffy728** [reply]
Marked as answer!

**Reply by Fluffy728** [reply]
Thanks
